### PR TITLE
Datasets Conversion Support and Fixing Bugs

### PR DESCRIPTION
**Pull request by wtomin**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

1. README.md and README_CN.md, increases the `Dataset` section, introducing the method to download and convert datasets;
2. fix some link errors:
3. fix convert_datasets.sh missing `fi` problem;
4. fix IaaAugment bug;
5. fix tools/dataset_converters/totaltext.py bug

### CODE REVIEW

**Review by SamitHuang — APPROVED** [review]

File: mindocr/data/transforms/iaa_augment.py [hunk 1]
```diff
@@ -21,5 +19,6 @@ def __call__(self, data):
                 new_polys.append(np.array([[kp.x, kp.y] for kp in kps.keypoints]))
 
             data['polys'] = np.array(new_polys) if isinstance(data['polys'], np.ndarray) else new_polys
+        data['image'] = aug.augment_image(data['image'])
```
Comment (zhtmike) [inline mindocr/data/transforms/iaa_augment.py:22]: 想问下aug.augment_image摆前后会有什么影响？

File: mindocr/data/transforms/det_transforms.py [hunk 2]
```diff
@@ -101,6 +101,7 @@ def __call__(self, data):
 
     def _draw_border(self, np_poly, border, mask):
         # draw mask
+        np_poly = copy(np_poly)
```
Comment (hadipash) [inline mindocr/data/transforms/det_transforms.py:104]: There's no need in explicit `copy`. This bug is fixed in [PR#204](https://github.com/mindspore-lab/mindocr/pull/204/files#diff-f2f94e4fbc850487c559769e5ca108c84f37d7f91c970e17a04e9b8cdc017836R110) but not merged yet.

File: docs/cn/datasets/icdar2015_CN.md [hunk 3]
```diff
@@ -8,8 +8,29 @@ ICDAR 2015 [文章](https://rrc.cvc.uab.es/?ch=4)
...
+ICDAR 2015 挑战赛分为三个任务。任务1是文本定位。任务3是单词识别。任务4是端到端文本检测识别。任务2文本分割的数据不可用。
+
+### Text Localization
+
+这四个文件分别是 ch4_training_images.zip、ch4_training_localization_transcription_gt.zip、ch4_test_images.zip 和 Challenge4_Test_Task1_GT.zip。它们的下载链接如下所示。
+
+
+Training Set
+
+[Training Set Images (88.5MB)](https://rrc.cvc.uab.es/?com=downloads&action=download&ch=4&f=aHR0cHM6Ly9ycmMuY3ZjLnVhYi5lcy8/Y29tPWRvd25sb2FkcyZhY3Rpb249ZG93bmxvYWQmZmlsZT1jaDRfdHJhaW5pbmdfaW1hZ2VzLnppcA==).- 1000 images obtained with wearable cameras
```
Comment (hadipash) [inline docs/cn/datasets/icdar2015_CN.md:20]: This link contains your individual credentials and will not work on other machines. You need to replace it with the common download link: https://rrc.cvc.uab.es/?ch=4&com=downloads
The same applies to all datasets listed here.

**Review by hadipash — CHANGES_REQUESTED** [review]

File: mindocr/data/transforms/iaa_augment.py (hunk 1 above)
Comment (wtomin) [reply mindocr/data/transforms/iaa_augment.py:22]: 因为 `kps = imgaug.KeypointsOnImage([imgaug.Keypoint(p[0], p[1]) for p in poly], shape=data['image'].shape)` 是把`data['polys']` map 到 original image shape (before image augmentation).

但是aug.augment_image有可能会改变image shape, 摆在前面会造成kps 经过mapping后的位置不正确。是很隐蔽的错误。

File: docs/cn/datasets/icdar2015_CN.md (hunk 3 above)
Comment (wtomin) [reply docs/cn/datasets/icdar2015_CN.md:20]: I will correct them. Thanks!

File: mindocr/data/transforms/iaa_augment.py (hunk 1 above)
Comment (zhtmike) [reply mindocr/data/transforms/iaa_augment.py:22]: 明白

File: mindocr/data/transforms/det_transforms.py (hunk 2 above)
Comment (wtomin) [reply mindocr/data/transforms/det_transforms.py:104]: I notice that `data['polys']` has been changed in `self._draw_border`, but I didn't identify which line caused it. 

If you confirm that `np_poly -= min_vals` is the line, I will remove this change.

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**